subsys/bluetooth/mesh/access.c

```diff
@@ -855,6 +873,39 @@ static int pub_period_start(struct bt_mesh_model_pub *pub)
 	return 0;
 }
 
+static uint8_t pub_delay_get(int random_delay_window)
+{
+	if (!IS_ENABLED(CONFIG_BT_MESH_DELAYABLE_PUBLICATION)) {
+		return 0;
+	}
+
+	uint8_t num = 0;
```

Comment:
Hm... whether `uint8_t` type wide is sufficient to represent random value above 256? Shouldn't it be `uint16_t`?

Reply:
but function still returns `uint8_t` value. It is still not possible to gave random values in range `256 ... 500`.